### PR TITLE
fix(snyk): use tag and not name of image to scan. fix naming

### PR DESCRIPTION
**Pull request by XSlemX**
- fixes names for jobs in reusables that were copy/pasted
- passes tag name to snyk job to allow container scanning of correct tag (altthough I assume it uses latest anyways)

### CODE REVIEW

File: .github/workflows/reusable_report-junit.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ on:
         description: 'SHA256 of the published image'
         required: true
 jobs:
-  pull-request:
+  j-unit:
```
Comment (AuStien) [inline .github/workflows/reusable_report-junit.yaml:26]: Maybe this should be called `report-j-unit`, for consistency's sake?
Also, see https://github.com/stacc/github-workflow-actions/issues/13

Comment (XSlemX) [reply .github/workflows/reusable_report-junit.yaml:26]: Agreed.

**Review by AuStien — APPROVED** [review]
:100:

**Review by maulik13 — APPROVED** [review]
LGTM

**Pull request merged**